cli/flags.rs

```diff
@@ -172,6 +174,10 @@ To get help on the another subcommands (run in this case):
     ).arg(
       Arg::with_name("reload")
         .short("r")
+        .min_values(0)
+        .takes_value(true)
+        .use_delimiter(true)
+        .require_equals(true)
         .long("reload")
         .help("Reload source code cache (recompile TypeScript)")
```

Comment:
Please add note that blacklist is supported and also add `long_help` with a few examples.